playbooks/ci-ceph_swift/templates/usr/bin/lshw

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+hostname="{{ inventory_hostname }}"
+
+{% raw -%}
+# run the original lshw command
+if [ "$1" != "-C" -o "$2" != "disk" ]; then
+  if [ -f /usr/bin/lshw.original ]; then
+    sudo /usr/bin/lshw.original
```

Comment:
Also, if we're going to use the original lshw, shouldn't we exit at this point, so we don't add extra data?